lib/helpers.bash

```diff
@@ -48,6 +48,19 @@ function _completion_exists ()
   complete -p "$1" &> /dev/null && _log_warning "$msg" ;
 }
 
+function _bash_it_homebrew_check()
+{
+	if [[ "${BASH_IT_HOMEBREW_PREFIX:-unset}" == 'unset' ]]
```

Comment:
the thing is that after this change, if someone installs/removes brew, he have to restart his shell in order for this to properly function. maybe we should unset this variable when calling `bash-it reload`, and also always check if the binary exists, even if the prefix is set (and unset it on failure)

Reply:
If `brew` is added to `$PATH` after shell startup (e.g., user adds it), and then `bash-it reload`s, then it will be picked up. But, if the user *removes* it then as written this will act as though it still is. I think this is very unlikely, but it's easy enough to add an `else` to unset it.